src/main/python/dha/cli.py

```diff
@@ -0,0 +1,345 @@
+        image['pull_count'] = int(image['pull_count'])
+    return imageDictList
+
+
+def filter_tag_list(tagDictList):
+    # Scrape out the results from the the MP work:
+    utc = pytz.UTC
+    justTagsList = []
+    for item in tagDictList:
+        justTagsList.extend(item['result'])
+
+    for tag in justTagsList:
+        tag.pop("last_updater", None)
+        tag.pop("v2", None)
+        tag.pop("id", None)
+        tag.pop("creator", None)
+        tag.pop("image_id", None)
+        tag.pop("repository", None)
+        tag['architecture'] = tag['images'][0]['architecture']
+        tag['os'] = tag['images'][0]['os']
+        tag.pop("images", None)
+        tag['tag_name'] = tag['name']
+        tag.pop('name', None)
+        tag['size_in_MB'] = round(tag['full_size'] / 1024 / 1024, 2)
+        tag.pop('full_size', None)
+        tag['days_since_update'] = (utc.localize(datetime.utcnow()) - dateutil.parser.isoparse(tag['last_updated'])).days
+    return justTagsList
+
+
+def check_result(process_data):
+    """ raise exception if any result in process data is exception
+    """
+    if any([isinstance(process.get('result'), Exception) for process in process_data]):
+        raise Exception('one or more processes had errors')
+
+
+def get_all_tags(client, function, args, imageDictList):
+    imageTagsDictList = []
+    screen_layout = None
+    if args.screen:
+        screen_layout = get_screen_layout()
+
+    execute(
+        function=function,
+        process_data=imageDictList,
+        shared_data={
+            'client': client,
+            'args': args,
+            'imageTagsDictList': imageTagsDictList
+        },
+        number_of_processes=args.processes,
+        init_messages=[
+            f"Total Image Count:{len(imageDictList)}",
+            f"Started:{datetime.now().strftime('%m/%d/%Y %H:%M:%S')}"
+        ],
+        screen_layout=screen_layout)
+
+    check_result(imageTagsDictList)
+
+    return imageDictList
+
+
+@queue_handler
+def get_tags(image, shared_data):
+    client = shared_data['client']
+    args = shared_data['args']
+    imageTagsDictList = shared_data['imageTagsDictList']
+    logger.debug('Fetching information about: {}'.format(image['name']))
+    response = client.get('/repositories/{}/{}/tags/?page_size=1000'.
+                          format(args.docker_user, image['name']))
+    for tag in response['results']:
+        tag['repo_name'] = image['name']
+        tag['repo_star_count'] = image['star_count']
+        tag['repo_pull_count'] = image['pull_count']
+        imageTagsDictList.append(tag)
+        logger.debug('Tag Processed')
+    return imageTagsDictList
+
+
+def main():
+    """ main function
+    """
+    parser = get_parser()
+    try:
+        args = parser.parse_args()
+        validate(args)
+
+        configure_logging(args)
+
+        dockerhub_client = get_dockerhub_client(args.dockerhub_host_api)
+
+        imageDictList = get_all_images(args, dockerhub_client)
```

Comment:
code leverages competing variable name styles; `lower_case` vs `mixedCase`
ideal to choose one case throughout.
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
This goes back to a standard question we have yet to address as formal guidance.